cogs/helpermessage.py

```diff
@@ -0,0 +1,149 @@
+import discord
+import toml
+from discord.ext import commands
+from utils.commands import available_subcommands
+from utils.pagination import paginated_embed_menus, PaginationView
+from db.models import HelperMessage
+
+class Helpermessage(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.data = toml.load('config.toml')
+        self.subjects = {
+            channel: item if isinstance(item, list) else [item]
+            for channel, item in self.data['subjects'].items()
+        }
+        self.description = self.data['description']
+        self.helpermessages = self.data['helpermessages']
+        self.subject_channels = self.subjects.keys()
+        self.helper_roles = [self.subjects[channel] for channel in self.subject_channels]
+
+    @commands.hybrid_group()
+    async def helpermessage(self, ctx: commands.Context):
+        """
+        Commands for handling reminders.
+        """
+        await available_subcommands(ctx)
+
+    @helpermessage.command()
+    async def list(self, ctx: commands.Context):
+        """
+        List all active helpermessage embeds
+        """
+        embed_dict = dict(
+            title = 'List of all active helpermessages',
+            description = 'Here is a list of all active helpermessages.',
+        )
+        names = [f'<#{channel}>' for channel in self.helpermessages.keys()]
+        values = [f'https://discord.com/channels/{ctx.guild.id}/{channel}/{int(self.helpermessages[channel])}' for channel in self.helpermessages.keys()]
+        embeds = paginated_embed_menus(names, values, embed_dict=embed_dict)
+        embed, view = await PaginationView(ctx, embeds).return_paginated_embed_view()
+        await ctx.send(embed=embed, view=view)
+
+    async def embed_getter(self, edited_roles):
+        for role in edited_roles:
+            for channel, roles in self.subjects.items():
+                if role.id in roles:
+                    discord_channel = await self.bot.fetch_channel(int(channel))
+                    helpermessage = await discord_channel.fetch_message(int(self.helpermessages[channel]))
+                    embed = helpermessage.embeds[0]
+                    yield helpermessage, role, embed
+
+    @commands.Cog.listener()
+    async def on_member_update(self, before: discord.Member, after: discord.Member):
```

Comment:
When you're creating a new embed, make sure you're adding fields with `inline = False` just like with helpermessage creation